blueocean-dashboard/src/main/js/components/PageLoading.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Progress } from '@jenkins-cd/design-language';
+
+/**
+ * Simple component to render a loading animation
+ */
+export default class PageLoading extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            percentage: undefined,
+            running: false,
+        };
+    }
+    
+    componentWillMount() {
+        if (this.props.duration) {
```

Comment:
yes, total length of _estimated_ time. it still goes away when not needed. rather than showing the unknown time animation, which looks bad